packages/util/src/bytes.ts

```diff
@@ -279,7 +279,16 @@ export const baToJSON = function (ba: any): any {
 
 /**
  * Checks provided Buffers for leading zeroes and throws if found.
+ *
+ * Examples:
+ *
+ * Valid values: 0x1, 0x, 0x01, 0x1234
+ * Invalid values: 0x0, 0x00, 0x001, 0x0001
+ *
+ * Note: This method is useful for validating that RLP encoded integers comply with the rule that all
+ * integer values encoded to RLP must be in the most compact form and contain no leading zeroes
  * @param values An object containing string keys and Buffer values
+ * @throws if any provided value is found to have leading zeroes
```

Comment:
This comment is not entirely correct, as `0x01` is valid; should be something like "throws if value has any (partial) leading zero bytes"

Reply:
Good call.  I'll update once I validate this on the R&D discord.